lib/otf/fontsense.cpp

```diff
@@ -0,0 +1,159 @@
+#include "fontsense.h"
+#include "unimath/uni_font.h"
+
+#include <stdlib.h>
+#include <queue>
+#include <string.h>
+#include <sstream>
+#include <filesystem>
+#include <optional>
+
+namespace fs = std::filesystem;
+
+#define FONTDIR "tinytex"
+
+namespace tinytex {
+
+font_paths_t get_font_paths() {
+	std::queue<std::string> paths;
```

Comment:
This project prefers 2 spaces indention, recommend following clang-format-style:

```
{
    BasedOnStyle: Google,
    IndentWidth: 2,
    AccessModifierOffset: -2,
    AllowShortBlocksOnASingleLine: true,
    AllowShortIfStatementsOnASingleLine: true,
    AllowShortCaseLabelsOnASingleLine: true,
    AllowShortLoopsOnASingleLine: true,
    ColumnLimit: 0,
    AlignAfterOpenBracket: false,
    BinPackArguments: false,
    BinPackParameters: true,
    AllowAllParametersOfDeclarationOnNextLine: true,
    ReflowComments: true,
    AlignConsecutiveMacros: false,
    AlignTrailingComments: true,
    AlignConsecutiveAssignments: false,
    AlignConsecutiveDeclarations: false,
    ContinuationIndentWidth: 2
}
```

I'll make a contribution guide ASAP.

Reply:
We can add a .clang-format file to the project.